public/views/question.html

```diff
@@ -1,38 +1,39 @@
 <div id="question-container-inner">
-  <div class="card longBlack">
-    <span ng-bind-html-unsafe="game.curQuestion.text" ng-hide="game.state === 'game ended' || game.state ==='game dissolved' || game.state === 'awaiting players'"></span>
+    <div class="card longBlack">
+        <span ng-bind-html-unsafe="game.curQuestion.text" ng-hide="game.state === 'game ended' || game.state ==='game dissolved' || game.state === 'awaiting players'"></span>
```

Comment:
2 spaces not 4 for any HTML file. Therefore, the spacing for this file needs fixing.
Also, break the lines that are too long to use multiline attributes as illustrated in previous comments